app/src/main/java/com/example/printer/settings/SettingsScreen.kt

```diff
@@ -0,0 +1,1107 @@
+            contentPadding = PaddingValues(vertical = 16.dp)
+        ) {
+            item {
+                // Printer Information Card
+                Card(
+                    modifier = Modifier.fillMaxWidth(),
+                    elevation = CardDefaults.cardElevation(defaultElevation = 2.dp)
+                ) {
+                    Column(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(16.dp)
+                    ) {
+                        Text(
+                            text = "Printer Information",
+                            style = MaterialTheme.typography.titleMedium
+                        )
+                        Spacer(modifier = Modifier.height(16.dp))
+                        
+                        if (isEditingName) {
+                            OutlinedTextField(
+                                value = tempPrinterName,
+                                onValueChange = { tempPrinterName = it },
+                                label = { Text("Printer Name") },
+                                modifier = Modifier.fillMaxWidth(),
+                                singleLine = true,
+                                trailingIcon = {
+                                    IconButton(onClick = {
+                                        if (tempPrinterName.isNotBlank()) {
+                                            printerName = tempPrinterName
+                                            PreferenceUtils.saveCustomPrinterName(context, tempPrinterName)
+                                            isEditingName = false
+                                        }
+                                    }) {
+                                        Icon(
+                                            imageVector = Icons.Default.Check,
+                                            contentDescription = "Save"
+                                        )
+                                    }
+                                }
+                            )
+                        } else {
+                            Row(
+                                modifier = Modifier.fillMaxWidth(),
+                                horizontalArrangement = Arrangement.SpaceBetween,
+                                verticalAlignment = Alignment.CenterVertically
+                            ) {
+                                Text("Name: $printerName")
+                                TextButton(onClick = {
+                                    tempPrinterName = printerName
+                                    isEditingName = true
+                                }) {
+                                    Text("Edit")
+                                }
+                            }
+                        }
+                        
+                        Spacer(modifier = Modifier.height(8.dp))
+                        Text("Status: Running") 
```

Comment:
Since this is hardcoded, does it actually mean anything?  I would suggest to leave it out if it doesn't have any other states.